scripts/copyAssets.js

```diff
@@ -1,6 +1,8 @@
-/* eslint-disable import/no-extraneous-dependencies, no-console */
```

Comment:
Renamed this file because of tab autocompletion... I would stuck at `vim ./script/build` when trying to edit `build.js` from terminal.